utils/install-operator-sdk.sh

```diff
@@ -5,7 +5,7 @@
 
 set -euo pipefail
 
-ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n "$(uname -m)" ;; esac)
+case $(uname -m) in x86_64|aarch64) ARCH="amd64";; *) ARCH="$(uname -m)";; esac
```

Comment:
fyi, I'm going to add a CI test to make sure these work on both macOS and Linux. wip here https://github.com/Kuadrant/kuadrant-operator/runs/4846610978?check_suite_focus=true

Couple of other issues with the script i noticed, will fix it up and update it across the repos it's been added to.